recipes/chemcoord/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "chemcoord" %}
+{% set version = "2.0.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/chemcoord-{{ version }}.tar.gz
+  sha256: 232d4a5813ec86adf5c0d86940ca345b717b3e68f003aa7925d6f1361192b1e4
+  patches:
+    - readme.patch
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
```

Comment:
should this be `build`, or is `host` correct?

Reply:
That part was from `grayskull` but host makes sense:
- pip is always required for host
- everything requires python >=3.6